src/gz_TEST.cc

```diff
@@ -135,7 +135,7 @@ TEST(CmdLine, GazeboServer)
 TEST(CmdLine, SimtimeArgument)
 {
   std::string cmd =
-    kGzCommand + " -r -v 4 --iterations 100 --initial-sim-time 1000.5 " +
+    kGzCommand + " -r -v 4 --iterations 500 --initial-sim-time 1000.5 " +
```

Comment:
Since the loaded sdf file has some plugins, none of the more time consuming plugins such as Physics are loaded. Also, the `plugins.sdf` world is set to run as fast as possible, so this command may finish running before transport is done initializing. My suggestion is to slow it down using `-z 1000` or even `-z 100`.

Reply:
https://github.com/gazebosim/gz-sim/pull/1916/commits/d23827458ce9491dda2b7d7e586c8a4485161948